python/analysis/auto_generated/raster/qgsninecellfilter.sip.in

```diff
@@ -34,7 +35,8 @@ Starts the calculation, reads from mInputFile and stores the result in mOutputFi
 
 :param feedback: feedback object that receives update and that is checked for cancelation.
 
-:return: 0 in case of success*
+:return: 0 in case of success
+         TODO: return an enum
```

Comment:
For when is this todo planned?

Reply:
No idea, it is not related to this PR and I've not the necessary kowledge to to do it quickly. I just wanted to note it. I'll remove the TODO if you prefer.

Reply:
Not really.
I just wondered if it's API related and therefore the point in time is QGIS 4 (in which case I think it should be tagged appropriately and can remain in the API docs) or who will take care of this at what point in time. In the second case I think there's no point in tagging this in doc-visible way.

Reply:
That code is API and it is quite old, I think we should file an issue for QGIS 4 to fix this, and I didn't really want to make it land in the docs, I just wanted to note that it is something that should be eventually fixed. BTW, there is no functional change but  it's just code style to return a human-readable meaningful enum instead of a meaningless int.